### DISCUSSION TITLE
Why isn't an ISIN validating?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [ghostfolio/ghostfolio#3749] [Q&A] [answered]
I'm trying to add https://markets.ft.com/data/funds/tearsheet/summary?s=GB00BD940N66:GBP to https://ghostfol.io/en/portfolio/activities?createDialog=true by entering "`GB00BD940N66`" into the ISIN field. However, the surrounding field border is red, and the "Done" button is disabled. Nothing else appears to validate either, despite the field informing the user that it accepts names too.

**Comment by Dilosen** [comment]
give it some time, it's a bit slow with the look up and sometimes can be brutally slow with tickers. What I do is restart the application, or if self hosted, just do a quicker restart of the container. That should speed things up and you can try again.

**Reply by roke-julian-lockhart** [reply]
@Dilosen, I'm using the (presumably official) https://ghostfol.io/ (as aforementioned) so is there anything I can actually do except wait (since I don't control the hosting)? I dare say that such a bug rather renders it useless, surely?

**Reply by dtslvr** [reply, marked as answer]
Hi @RokeJulianLockhart

Unfortunately, the mentioned fund “OneFamily Global Equity Fund” is not supported by our data providers.

**Reply by roke-julian-lockhart** [reply]
@dtslvr, could I file a feature request for it? I'm not sure what to title it, though.

**Reply by dtslvr** [reply]
Good news! One of our professional data providers has successfully enabled `GB00BD940N66`, now available in [Ghostfolio Premium](https://ghostfol.io/en/pricing).
